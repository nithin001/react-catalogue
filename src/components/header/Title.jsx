import React from 'react';
import { Link } from 'react-router-dom';
const Title = props => (
  <Link className={'title left brand-logo'} to="/">Acme Tech Shop</Link>);

export default Title;
