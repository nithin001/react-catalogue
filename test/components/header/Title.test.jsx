import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Title from '../../../src/components/header/Title';

describe('title component', () => {
  it('should render the title component', () => {
    const title = renderer.create(<MemoryRouter><Title/></MemoryRouter>);
    expect(title).toMatchSnapshot();
  });
});