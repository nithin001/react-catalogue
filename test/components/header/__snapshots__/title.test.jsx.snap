// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`title component should render the title component 1`] = `
<a
  className="title left brand-logo"
  href="/"
  onClick={[Function]}
>
  Acme Tech Shop
</a>
`;
